hs_core/hydroshare/utils.py

```diff
@@ -428,58 +416,38 @@ def copy_resource_files_and_AVUs(src_res_id, dest_res_id, set_to_private=False):
     avu_list = ['bag_modified', 'metadata_dirty', 'isPublic', 'resourceType']
     src_res = get_resource_by_shortkey(src_res_id)
     tgt_res = get_resource_by_shortkey(dest_res_id)
+
+    # This makes the assumption that the destination is in the same exact zone.
+    # Also, bags and similar attached files are not copied.
     istorage = src_res.get_irods_storage()
     src_coll = os.path.join(src_res.root_path, 'data')
-    if src_res.resource_federation_path:
-        dest_coll = os.path.join(src_res.resource_federation_path, dest_res_id)
-        dest_coll += '/'
-    else:
-        dest_coll = dest_res_id + '/'
+    dest_coll = os.path.join(tgt_res.root_path, 'data')
 
+    # TODO: This copies everything but bags, including unmapped junk files(!)
     istorage.copyFiles(src_coll, dest_coll)
```

Comment:
@alvacouch Addressing your TODO comment, this is intended, we should not copy bags and manifest files since the user may changes things around after the copy action, then the bags and manifest files need to be regenerated anyway, so there is no point to copy the bag and manifest files over.